web/client/themes/default/less/widget.less

```diff
@@ -154,6 +154,57 @@
             margin-left: 5%;
             font-style: italic;
         }
+
+        .widget-selector {
+            flex-grow: 1;
+            position: relative;
+            display: flex;
+            justify-content: center;
+
+            .Select {
+                flex-grow: 1;
+                .Select-control {
+                    margin: 0 auto;
+                    max-width: 60%;
+                    height: 28px;
+
+                    .Select-placeholder {
+                        line-height: 24px;
+                    }
+
+                    .Select-value {
+                        padding-right: 18px;
+                        padding-left: 5px;
+                        line-height: 24px !important;
+
+                        .Select-value-label {
+                            line-height: 24px;
+                        }
+                    }
+                    .Select-input {
+                        height: 24px;
+                    }
+                }
+                .Select-menu-outer {
+                    margin-top: 2px;
+                }
+            }
```

Comment:
Do we need all these changes on the select component?

Reply:
At least for the height, yes, widget height is 28px while react-select default style is 34px and in this version of react-select that's the only way to control its height (by rewriting it in a few places). Rest of styles are for its proper work with arrow navigation including `transform` property.